src/kata2.booleanToWord.js

```diff
@@ -1,5 +1,11 @@
-const booleanToWord = (boolean) => {
+// const booleanToWord = (boolean) => {
+//   if (boolean) {
+//     return 'Yes'
+//   }
+//     return 'No'
+// }
 
+const booleanToWord = (boolean) => {
+return (boolean ? "Yes" : "No");
```

Comment:
Indentation! Be sure to be consistent with quotes also. Choose either single quotes or double quotes, and stick to your choice.